### PR TITLE
Improved syslog regex parser configuration

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Give a better example that works!

# Impacts
admin guide doc

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -2934,6 +2936,30 @@ Now you will need to restart the pfqueue and the pfdetect services.
 
 Make sure that you have your pipe file otherwise the process won't start.
 
+Rsyslog configuration
+---------------------
+
+You will need to create a rsyslog configuration to forward all the syslog message sent by StreamScan to the pipe file /usr/local/pf/var/cds
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: all the syslog messages

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -2934,6 +2936,30 @@ Now you will need to restart the pfqueue and the pfdetect services.
...
+
+First you need to activate the syslog under the rsyslog configuration file located at /etc/rsyslog.conf, uncomment the two parameters:
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: activate -> enable

**Comment by dwlfrth** [comment]
@jrouzierinverse review and merge

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -2870,14 +2870,14 @@ Here is the configuration of the syslog regex parser:
 
     Detector *: StreamScan
     Enabled: checked
-    Alert pipe: /usr/local/pf/var/streamscan (To create the fifo file, do: mkfifo /usr/local/pf/var/streamscan)
+    Alert pipe: /usr/local/pf/var/cds (To create the fifo file, do: mkfifo /usr/local/pf/var/cds)
```
Comment (jrouzierinverse) [inline docs/PacketFence_Administration_Guide.asciidoc]: Put the creation of the fifo in the rsyslog configuration section to avoid confusion.

**Pull request merged**